src/main/java/uk/gov/ida/eventemitter/EventEmitterModule.java

```diff
@@ -38,7 +57,31 @@ private SqsClient getAmazonSqsClient(final Optional<AmazonSQS> amazonSqs,
     }
 
     @Provides
-    private EventEmitter getEventEmitter(final SqsClient sqsClient) {
-        return new EventEmitter(new StubEncrypter(), sqsClient);
+    private Encrypter getEncrypter(final Optional<AmazonS3> amazonS3,
+                                   final Optional<Configuration> configuration,
+                                   final ObjectMapper mapper) {
+        if (amazonS3.isPresent()) {
+            try {
+                S3Object o = amazonS3.get().getObject(configuration.get().getBucketName(), configuration.get().getKeyName());
+                S3ObjectInputStream s3is = o.getObjectContent();
```

Comment:
In general I'd rather avoid 1 letter characters and non trivial abbreviations, so would probably have `s3Object` and `s3InputStream` here, but that's mostly a personal preference.